SECURITY.md

```diff
@@ -0,0 +1,63 @@
+We are committed to conduct our Security Process in a professional and civil manner.
+Public shaming, under-reporting, or misrepresentation of vulnerabilities will not be tolerated.
+
+To submit a finding, please follow the steps outlined in receiving disclosures [section](#disclosing-a-vulnerability).
+
+## Responsible Disclosure Standard
+
+ApeWorX follows a community [standard](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#the-standard) for responsible disclosure in cryptocurrency and related software.
+This document is a public commitment to following the standard.
+
+This standard provides detailed information for:
+
+- [Initial Contact](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#initial-contact):
+  how to establish initial contact with our security team.
+- [Giving Details](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#giving-details):
+  what details to include with your vulnerability disclosure after having received a response to your initial contact.
+- [Setting Dates](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#setting-dates):
+  how to agree on timelines for releasing updates and making details of the issue public.
+
+Any expected deviations and necessary clarifications around the standard are explained in the following sections.
+
+## Disclosing a Vulnerability
+
+### Supported Versions
+
+Ape is currently under active development, targeting a 1.x release.
+Any security issues found on the latest minor version of Ape will be fixed and released in either a patch or minor release.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| > 0.5.x | :white_check_mark: |
+
+### Directly to ApeWorX
+
+ApeWorX is committed to working with researchers who submit security vulnerability notifications to us, to resolve those issues on an appropriate timeline, and to perform a coordinated release, giving credit to the reporter if they would so like.
+
+Please submit disclosures to **all** of the following main points of contact for security related issues according to the [initial contact](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#initial-contact) and [giving details](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#giving-details) guidelines.
+
+For all security related issues, ApeWorX has the following main points of contact:
+
+| Contact                                                | Keybase                                                      |
+| ------------------------------------------------------ | ------------------------------------------------------------ |
+| [@fubuloubu](https://github.com/fubuloubu)             | [@fubuloubu](https://keybase.io/fubuloubu/chat)              |
+| [@NotPeopling2day](https://github.com/NotPeopling2day) | [@NotPeopling2day](https://keybase.io/notepeopling2day/chat) |
```

Comment:
I feel left out. Do we need more people here? I volunteer if yes...

Reply:
Yes, but you need to add yourself to Keybase lol

Reply:
My username is `antazoey`!